eng/pipelines/stress-http.yml

```diff
@@ -0,0 +1,41 @@
+trigger: none
+
+pool:
+  name: Hosted VS2017
```

Comment:
Is this going to run on Windows only?

Reply:
No. This is just getting an MVP running. Will iterate.